src/HotChocolate/Data/test/HotChoclate.Data.NHibernate.Tests/HotChoclate.Data.NHibernate.Tests.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
@PascalSenn @michaelstaib we need to clean the project files and align them with the project.